includes/class-wc-gateway-stripe.php

```diff
@@ -874,7 +884,7 @@ public function prepare_order_pay_page( $gateways ) {
 	 * @return string the new message.
 	 */
 	public function change_no_available_methods_message() {
-		return wpautop( __( "Almost there!\n\nYour order has already been created, the only thing that still needs to be done is for you to authorize the payment with your bank.", 'woocommerce-gateway-stripe' ) );
```

Comment:
Why did you remove the `wpautop`? It seems like this is being rendered inside a `<li>`, is that the reason why there shouldn't be `<p>`? In that case, the `\n\n` won't render at all, because this is plain HTML, right?

Reply:
The HTML is escaped in the template used for this screen [[source](1405-gh-woocommerce/woocommerce-subscriptions/blob/080fb2e86cb088f02c6a8212d802330baea309c9/templates/checkout/form-change-payment-method.php#L82)], so I thought it might be acceptable to just omit the paragraph break. (Noted in the https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1405/commits/df281efca560ac10b02ea0f6362fe429cdf3e9d1 message.)

I kept them in the string thinking it's not worth impacting the translation – what do you think?

(Taking a step back, another viable approach for the authorization screen might have been to keep Stripe as the one available gateway, and just modify the payment fields output.)

Reply:
> I kept them in the string thinking it's not worth impacting the translation – what do you think?

That's fair, I hadn't read the commit message, I'm fine with that change 👍